Source/ACE.Server/WorldObjects/Door.cs

```diff
@@ -77,6 +77,26 @@ public string LockCode
             set { if (value == null) RemoveProperty(PropertyString.LockCode); else SetProperty(PropertyString.LockCode, value); }
         }
 
+        public override void OnActivate(WorldObject activator)
```

Comment:
this override only includes a small subset of the logic in WorldObject_Use.OnActivate(), and does not call the base

all of the existing / very important logic in WorldObject_Use.OnActivate() suddenly gets thrown out with this change. For the logic it does include, it also duplicates from the base

the idea is for the base to do everything that needs to be done for all objects, and then for the overrides to build on top of that.

i'm not sure what all of the exact requirements are for doors yet, but it seems like this has the potential to break many other door activations, without careful review of all the data first. for example, how is the ""Beyond lays the second test" message sent to the user, is that a Talk or Emote response?

This code seems to assume that doors should only ever have an ActivationResponse.Use, and that it always executes 100% of the time. If that is the case, why not just add ActivationResponse |= ActivationResponse.Use to SetEphemeralValues?